src/dynamo.py

```diff
@@ -61,6 +62,44 @@ def _scan_with_condition(
     return items
 
 
+def is_receiver_available(receiver: str):
+    """Check if receiver is available"""
+    items: List[GemsModel] = _scan_with_condition(
```

Comment:
does this have to be a scan?

Reply:
I quickly opted for the `_scan_with_condition` since it was already available for other operations. Ideally, we can (and should) change the DDB schema to fully utilize the benefits of PK and SKs.

Reply:
should we do that now?
this will deffo slow down the performance